docs/components/concepts/process-instance-migration.md

```diff
@@ -332,11 +340,11 @@ If your specific case is not (yet) supported by process instance migration, you
 Note that this results in new keys for the process instance and its associated variables, element instances, and other entities.
 :::
 
-### Supported BPMN elements
+## Supported BPMN elements
```

Comment:
🔧 IMO, this should be part of the limitations section. While we take the positive approach of documented what is supported, it actually describes that there are elements that cannot be migrated yet.